.changeset/cyan-parents-relax.md

```diff
@@ -0,0 +1,9 @@
+---
+'@credo-ts/anoncreds': patch
+'@credo-ts/askar': patch
+'@credo-ts/core': patch
+---
+
+- Remove usage of Big Number libraries and rely on native implementations
+- By default rely on uncompressed keys instead of compressed (for P256, P384, P521 and K256)
+- Utilze Uint8Array more instead of Buffer (i.e. for internally representing a key)
```

Comment:
These two should be marked as minor (due to breaking changes)

Reply:
Ah, will do.